Remove popup API WPT from tree

nt1m opened the pull request:
#### aad0f1a90bb69f52e61facd43500c5399783531d
<pre>
Remove popup API WPT from tree
<a href="https://bugs.webkit.org/show_bug.cgi?id=248898">https://bugs.webkit.org/show_bug.cgi?id=248898</a>

Reviewed by Chris Dumez.

We don&apos;t support this API, and also the tests are pretty out-of-date, the API is now called popover, and is now in the popovers folder on the WPT repo.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-anchor-idl-property.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-anchor-idl-property.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-anchor-nesting.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-anchor-nesting.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-display-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-cleanup.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-cleanup.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-display.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-display.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-finishes-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-finishes.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-finishes.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-show-display.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-show-display.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animation-corner-cases.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animation-corner-cases.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-appearance-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-appearance.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-appearance.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-attribute-basic.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-attribute-basic.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-attribute-unscopable.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-attribute-unscopable.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-backdrop-appearance-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-backdrop-appearance.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-backdrop-appearance.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-css-properties.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-css-properties.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-2.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-2.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-display-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-display.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-display.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-hints.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-hints.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-events.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-events.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-focus.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-focus.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-hidden-display-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-hidden-display.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-hidden-display.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-hoverpopup-attribute.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-hoverpopup-attribute.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-inside-display-none-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-inside-display-none.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-inside-display-none.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-invoking-attribute.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-invoking-attribute.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-light-dismiss-on-scroll.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-light-dismiss-on-scroll.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-light-dismiss.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-light-dismiss.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-not-keyboard-focusable.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-not-keyboard-focusable.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-open-display-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-open-display.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-open-display.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-open-overflow-display-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-open-overflow-display.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-open-overflow-display.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-removal-2.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-removal-2.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-removal.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-removal.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-shadow-dom.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-shadow-dom.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-show-event.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-show-event.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-stacking-context-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-stacking-context.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-stacking-context.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-stacking.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-stacking.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-top-layer-interactions.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-top-layer-interactions.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-types.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-types.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/resources/popup-styles.css: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/resources/popup-utils.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/resources/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/w3c-import.log: Removed.
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/257500@main">https://commits.webkit.org/257500@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/a9fe0a3d6faa5b0ecc1421fc0bd1df6257ae4f01

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [  ~~🧪 style~~](https://ews-build.webkit.org/#/builders/6/builds/99193 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🛠 ios~~](https://ews-build.webkit.org/#/builders/77/builds/8402 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac~~](https://ews-build.webkit.org/#/builders/43/builds/32328 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/8/builds/108586 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 🧪 win~~](https://ews-build.webkit.org/#/builders/10/builds/168833 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [  ~~🛠 ios-sim~~](https://ews-build.webkit.org/#/builders/76/builds/8952 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/61/builds/85727 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/91696 "Build is in progress. Recent messages:configuring build; Pull request 7278 is already closed") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/106521 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/104938 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/76/builds/8952 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/43/builds/32328 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/36/builds/91696 "Build is in progress. Recent messages:configuring build; Pull request 7278 is already closed") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/76/builds/8952 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/43/builds/32328 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 api-gtk](https://ews-build.webkit.org/#/builders/36/builds/91696 "Build is in progress. Recent messages:configuring build; Pull request 7278 is already closed") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/81/builds/2284 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/43/builds/32328 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/82/builds/2176 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/61/builds/85727 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/80/builds/7228 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/43/builds/32328 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/2623 "Built successfully and passed tests") | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/79/builds/3654 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

cdumez reviewed (changes requested):
We should just skip them.

cdumez reviewed (approved):
OK since html/semantics/popups no longer exists upstream. This is essentially a partial re-sync.

nt1m committed:
Remove popup API WPT from tree
https://bugs.webkit.org/show_bug.cgi?id=248898

Reviewed by Chris Dumez.

We don't support this API, and also the tests are pretty out-of-date, the API is now called popover, and is now in the popovers folder on the WPT repo.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-anchor-idl-property.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-anchor-idl-property.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-anchor-nesting.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-anchor-nesting.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-display-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-cleanup.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-cleanup.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-display.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-display.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-finishes-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-finishes.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-hide-finishes.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-show-display.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animated-show-display.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animation-corner-cases.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-animation-corner-cases.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-appearance-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-appearance.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-appearance.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-attribute-basic.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-attribute-basic.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-attribute-unscopable.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-attribute-unscopable.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-backdrop-appearance-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-backdrop-appearance.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-backdrop-appearance.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-css-properties.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-css-properties.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-2.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-2.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-display-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-display.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-display.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-hints.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen-hints.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-defaultopen.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-events.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-events.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-focus.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-focus.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-hidden-display-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-hidden-display.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-hidden-display.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-hoverpopup-attribute.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-hoverpopup-attribute.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-inside-display-none-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-inside-display-none.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-inside-display-none.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-invoking-attribute.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-invoking-attribute.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-light-dismiss-on-scroll.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-light-dismiss-on-scroll.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-light-dismiss.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-light-dismiss.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-not-keyboard-focusable.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-not-keyboard-focusable.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-open-display-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-open-display.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-open-display.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-open-overflow-display-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-open-overflow-display.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-open-overflow-display.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-removal-2.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-removal-2.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-removal.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-removal.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-shadow-dom.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-shadow-dom.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-show-event.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-show-event.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-stacking-context-ref.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-stacking-context.tentative-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-stacking-context.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-stacking.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-stacking.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-top-layer-interactions.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-top-layer-interactions.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-types.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/popup-types.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/resources/popup-styles.css: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/resources/popup-utils.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/resources/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popups/w3c-import.log: Removed.
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/257500@main
